contrib/vagrant/provision-master.sh

```diff
@@ -39,7 +37,9 @@ if [ "${SDN_NODE}" = "true" ]; then
 
   # Disable scheduling for the sdn node - it's purpose is only to ensure
   # pod network connectivity on the master.
-  os::provision::disable-sdn-node "${CONFIG_ROOT}" "${SDN_NODE_NAME}"
+  #
+  # This will be performed separately for dind.
```

Comment:
vagrant can be deployed in parallel but dind is effectively deployed serially for simplicity, so this speeds things up a bit by performing node provisioning before disabling the sdn node to give the sdn node time to register itself.